doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -1693,3 +1693,18 @@ The Solicitation PDU RPL Client model is an optional mesh subsystem that can be
 	* ``RngStart``: Start address of the SSRC range.
 	* ``Ackd``: This argument decides on whether an acknowledged or unacknowledged message will be sent.
 	* ``RngLen``: Range length for the SSRC addresses to be cleared from the solicitiation RPL list. This parameter is optional; if absent, only a single SSRC address will be cleared.
+
+
+Statistic
```

Comment:
Frame statistics?
Statistics for frame handling?

Reply:
Seems `frame statistic` is more relevant. I'll change it.